docs/tutorials/getting_started.ipynb

```diff
@@ -0,0 +1,209 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Getting started\n",
+    "\n",
+    "This tutorial uses safeds on **titanic passenger data** to predict who will survive and who will not, using sex and travel class as features for the prediction.\n"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "1. Load your data into a `Table`, the data is available under `docs/tutorials/data/titanic.csv`:\n"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "outputs": [],
+   "source": [
+    "from safeds.data.tabular.containers import Table\n",
+    "\n",
+    "titanic = Table.from_csv_file(\"data/titanic.csv\")"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "2. Use `OneHotEncoder` to create an encoder, that will be used later to transform the titanic table.\n",
+    "Use the `fit` function of the `OneHotEncoder` to pass the table and the column names, that will be used as features to predict who will survive to the encoder.\n",
+    "The names of the columns before transformation need to be saved, because `OneHotEncoder` will change the names of the fitted `Column`s:\n"
```

Comment:
you could also explain why you use the `OneHotEncoder`. You could say that you want the non-numerical values (that seem important for the prediction) to be represented by numbers

Reply:
Applied that too